init.sls

```diff
@@ -48,14 +48,19 @@ chown_pgdata:
       wal_log_hints: false
       max_wal_senders: 5 # NOTE: Since postgresql-10, the default is 10
       log_autovacuum_min_duration: '500ms'
+      log_statement: 'all'
+      log_directory: 'pg_log'
+      log_filename: 'postgresql-%Y-%m-%d_%H%M%S.log'
+      logging_collector: on
```

Comment:
`on` will be interpreted as `True` - therefore this will result in a broken `postgresql.conf`. 

I'm currently using `true` and `false` here in the configuration and then handle the setting in the configuration file like so:


```jinja
{%- if wal_log_hints %}
wal_log_hints = on
{%- else %}
wal_log_hints = off
{%- endif %}
```